spec/registry.md

```diff
@@ -489,6 +503,14 @@ MUST be multiplied by 100 for the purposes of formatting.
 
 The _function_ `:number` performs selection as described in [Number Selection](#number-selection) below.
 
+#### Composition
+
+When an _operand_ or an _option_ value uses a _variable_ annotated,
+directly or indirectly, by a `:number` _annotation_,
+its resolved value contains an implementation-defined numerical value
```

Comment:
Would it be more precise to say that the `getValue()` method of its resolved value returns an implementation-defined numeric value? (Assuming the definition of resolved values in #728.)

Reply:
Yes.